mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/dto/query/QueryDTO.java

```diff
@@ -65,6 +67,13 @@ public List<QueryElement> getCriteria() {
         return criteria;
     }
 
+
+    @JsonGetter("limit")
+    public Long getLimitAsLong() {
+        return limit.orElse(null);
+    }
+
+    @JsonIgnore
```

Comment:
Add the JDK 8 Jackson Module to the serializer instead?

Reply:
Can you explain more? 

`@JsonIgnore` just for ignoring the `Optional limit` 

> Add the JDK 8 Jackson Module 

Your mean is `new ObjectMapper().registerModule(new Jdk8Module());` ? , the `QueryElementSerializer` already receives the `ObjectMapper` bean in constructors, I don't think we should modify the bean